AzureMS/scripts/npc/2010001.js

```diff
@@ -109,20 +110,20 @@ function action(mode, type, sel) {
                 			}
 				}
 
-                		cm.askAvatar("Please choose your favorite lens.", ����);
+                		cm.askAvatar("Please choose your favorite Color.", ����);
 			break;
 			case 5: // Mix Dyeing
-            			cm.sendYesNo("Do you use mix dye? #r#e(May cause hair pauses that cannot be mixed dye.)");
+            			cm.sendYesNo("Are you sure you want to use Mixed Dyes? #r#e(Some Hairs cannot use Mixed Dyes.)");
```

Comment:
Similarly, I don't think `hair`/`mixed dyes` are supposed to be a proper nouns here. Consider lower-case?